src/Client.php

```diff
@@ -298,9 +320,18 @@ protected function paramsToQueryString(array $params = [])
             if (is_array($value)) {
                 // recursion case
```

Comment:
```suggestion
```

Reply:
I'm cool with that. I would find it cool, if we're not beautifying in this PR too much that has already been there in your code before. It just slows down this process due to the back-and-forth ... :-)